src/scripts/visualizations/legend.js

```diff
@@ -5,6 +5,7 @@
             vAlign: 'middle',
             hAlign: 'right',
             direction: 'vertical',
+            enabled: true,
```

Comment:
This property has been added to the defaults, not changed. Otherwise you would see a red line removing the previous value of this default (something like - enabled: false).

Possible that I may be misinterpreting.